src/PointwiseFunctions/Elasticity/ElasticPotentialEnergy.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Elliptic/Systems/Elasticity/Tags.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp"
+
+namespace Elasticity {
+
+template <size_t Dim>
+Scalar<DataVector> potential_energy_density(
+    const tnsr::ii<DataVector, Dim, Frame::Inertial>& strain,
+    const tnsr::I<DataVector, Dim>& coordinates,
+    const ConstitutiveRelations::ConstitutiveRelation<Dim>&
+        constitutive_relation) noexcept;
+
+namespace Tags {
+template <size_t Dim>
+struct PotentialEnergyCompute : Elasticity::Tags::PotentialEnergy<Dim>,
```

Comment:
Also add documentation to this class (you can refer to the function above as `\see Elasticity::potential_energy_density`)